.github/workflows/e2e.yml

```
@@ -0,0 +1,73 @@
+name: e2e
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)